### ISSUE TITLE
Duplicated autopilot parameters

### ISSUE THREAD

**Issue by rafaellehmkuhl** [bluerobotics/BlueOS-docker#1742]
![image](https://github.com/bluerobotics/BlueOS-docker/assets/6551040/01abdd8e-5fe7-4136-b3e2-93234f175f94)

I was using SITL and changed to Navigator. Don't know if this was the reason, though.

**Comment by Williangalvani** [comment]
interesting. we should probably empty the store when changing fw, though that doesn't quite explain the issue.. 🤔

**Comment by rafaellehmkuhl** [comment]
I think that explains the issue, no?

**Comment by Williangalvani** [comment]
> I think that explains the issue, no?

yeah, I just checked and the keys in ParametersTable are numbers, which allows for duplicated names.

**Comment by joaoantoniocardoso** [comment]
I couldn't reproduce it by changing from SITL to Pixhawk, and back to SITL again.

**Comment by rafaellehmkuhl** [comment]
> I couldn't reproduce it by changing from SITL to Pixhawk, and back to SITL again.

Strange...
I changed from SITL to Navigator, but unless there's some contamination between the Linux routines, this shouldn't make a difference.
